docs/installation/helm.md

```diff
@@ -1,75 +1,157 @@
 
-Before we proceed, let's ensure you have both KubeCTL and HelmCLI installed. 
+- Follow the instructions [here](https://minikube.sigs.k8s.io/docs/start/) to install Minikube using a package manager or the official installation guide.
 
-Open your terminal and check the versions of Docker and Helm with the following commands:
+### Step 2: Installing Docker, kubectl and Helm
 
-```
-$ docker version
-```
-Example Output:
-```
-Client:
- Cloud integration: v1.0.31
- Version: 20.10.24
-```
+Before proceeding, ensure you have both kubectl and Helm CLI installed. Open your terminal and check the versions of Docker, kubectl, and Helm:
 
+To install Docker, kubectl, and Helm on different operating systems, you can use the following commands:
+
+Certainly! Here are the commands to install Docker, kubectl, and Helm on Windows, MacOS, and Linux:
+
+<details>
+<summary>Windows</summary>
+
+#### Docker:
+Download and install Docker Desktop from the official Docker website: [Docker Desktop for Windows](https://www.docker.com/products/docker-desktop)
+
+#### kubectl:
+You can install kubectl on Windows using the following PowerShell command:
+
+```powershell
+curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.22.2/bin/windows/amd64/kubectl.exe
+Move-Item -Path .\kubectl.exe -Destination C:\Windows\System32\kubectl.exe
 ```
-$ helm version
-```
-Example Output:
+
+#### Helm:
+- Installing Helm on Windows
+- Go to the Helm releases page at https://github.com/helm/helm/releases.
+- Download the latest Windows executable file (ending with ".exe").
+- Double-click the downloaded file to start the installation wizard.
+- Follow the prompts to complete the installation.
+
+</details>
+
+<details>
+<summary>MacOS</summary>
+
+#### Docker:
+   
+Download and install Docker Desktop from the official Docker website: [Docker Desktop for Mac](https://www.docker.com/products/docker-desktop)
+
+#### kubectl:
+   
+You can install kubectl on MacOS using Homebrew:
+
+```bash
+brew install kubectl
 ```
-version.BuildInfo{Version:"v3.12.0", GitCommit:"c9f554d75773799f72ceef38c51210f1842a1dea", GitTreeState:"clean", GoVersion:"go1.20.4"}
+
+#### Helm:
+   
+You can install Helm on MacOS using Homebrew as well:
+
+1. Open a terminal window.
+
+Install Homebrew by running the following command:
+    
+ ```bash
+ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+ ```
+
+2. Once Homebrew is installed, run the following command to install Helm:
+   
+```bash
+brew install helm
 ```
 
-Please note these are recommended software versions, and your environment may have different versions.
+</details>
+
+<details>
+<summary>Linux</summary>
+
+#### Docker:
+Install Docker on Linux by following the official Docker installation guide for your distribution: [Docker Installation Guide](https://docs.docker.com/engine/install/)
```

Comment:
Put the steps in here.

Reply:
Use this page of the docs: https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository